contents/handbook/small-teams/pipeline/index.mdx

```diff
@@ -17,14 +17,14 @@ hideAnchor: false
 
 ## Responsibilities
 
-Team Ingestion owns our ingestion pipeline end-to-end. That means we own:
-- Django server ingestion API
```

Comment:
nit: we also own the delete, split APIs that users can click on from the app's person page